src/components/CheckIn/ItemsListModal.js

```diff
@@ -56,7 +56,7 @@ const ItemsListModal = ({
     'location': (item) => (
       <div className={css.fieldWrapper}>
         <Select
-          dataOptions={locations}
+          dataOptions={[{ label: '', value: '' }, ...locations]}
```

Comment:
seems we use it everywhere, mb we need smth generic?

Reply:
not everywhere, actually. Some dropdowns require empty option (optional fields), some of them not

Reply:
constant sounds good to me, I'm about it

Reply:
done